### ISSUE TITLE
BUG FOUND: DNS terraform failed to apply the TTL to the right value

### ISSUE THREAD

**Issue by georgedriver** [terraform-providers/terraform-provider-alicloud#796]
### Terraform Version
```
Terraform v0.11.11
+ provider.alicloud v1.31.0
```

### Affected Resource(s)
- alicloud_dns_record

### Terraform Configuration Files
```hcl
resource "alicloud_dns_record" "alicloud_dns_record-aliyundm-dev-platform-SPF" {
  host_record = "dev-platform"
  name        = "xxxxxxxx.xxx"
  priority    = 1
  routing     = "default"
  ttl         = 60
  type        = "TXT"
  value       = "<some value>"
}
```

### Debug Output
Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

### Panic Output
```
exit status 1: running "terraform apply -input=false -no-color \"/atlantis/repos/xxxxxxx/china-infrastructure-self-service/37/default/dns/alicloud/default.tfplan\"" in "/atlantis/repos/xxxxxxx/china-infrastructure-self-service/37/default/dns/alicloud"
alicloud_dns_record.alicloud_dns_record-aliyundm-dev-platform-OWNER: Modifying... (ID: 17196717320730624)
  ttl: "600" => "60"
alicloud_dns_record.alicloud_dns_record-aliyundm-dev-platform-SPF: Modifying... (ID: 17196717292848128)
  ttl: "600" => "60"

Error: Error applying plan:

2 error(s) occurred:

* alicloud_dns_record.alicloud_dns_record-aliyundm-dev-platform-SPF: 1 error(s) occurred:

* alicloud_dns_record.alicloud_dns_record-aliyundm-dev-platform-SPF: [ERROR] terraform-provider-alicloud/alicloud/resource_alicloud_dns_record.go:159: Resource 17196717292848128 UpdateDomainRecord Failed!!! [SDK alibaba-cloud-sdk-go ERROR]:
SDK.ServerError
ErrorCode: MissingRecordId
Recommend: https://error-center.aliyun.com/status/search?Keyword=MissingRecordId&source=PopGw
RequestId: A8D5C2ED-54B8-4BE3-B502-01DE50CE0A13
Message: RecordId is mandatory for this action.
* alicloud_dns_record.alicloud_dns_record-aliyundm-dev-platform-OWNER: 1 error(s) occurred:

* alicloud_dns_record.alicloud_dns_record-aliyundm-dev-platform-OWNER: [ERROR] terraform-provider-alicloud/alicloud/resource_alicloud_dns_record.go:159: Resource 17196717320730624 UpdateDomainRecord Failed!!! [SDK alibaba-cloud-sdk-go ERROR]:
SDK.ServerError
ErrorCode: MissingRecordId
Recommend: https://error-center.aliyun.com/status/search?Keyword=MissingRecordId&source=PopGw
RequestId: 0055CEE8-CC06-4C54-9930-24D3A84DBEBA
Message: RecordId is mandatory for this action.

Terraform does not automatically rollback in the face of errors.
Instead, your Terraform state file has been partially updated with
any resources that successfully completed. Please address the error
above and apply again to incrementally change your infrastructure.
```

### Expected Behavior
What should have happened?
Should update the TTL to the right value.

### Actual Behavior
What actually happened?
Panic.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

**Comment by andyzhang495** [comment]
[这里](https://github.com/terraform-providers/terraform-provider-alicloud/blob/02bcf11f79b31193a55615eba44baf4a59a88853/alicloud/resource_alicloud_dns_record.go#L79) 没有设置 request 的 `TTL` 属性，所以每次 Create 都是默认的 600 秒

**Comment by andyzhang495** [comment]
`MissingRecordId` 是 [这里](https://github.com/terraform-providers/terraform-provider-alicloud/blob/02bcf11f79b31193a55615eba44baf4a59a88853/alicloud/resource_alicloud_dns_record.go#L122) 的问题。把 `Id` 赋值到 `RegionId` 里面去了。编辑器自动补全的锅？

**Comment by andyzhang495** [comment]
相关 #640 
#640 中用户报告的问题其实并不是“设置了超出套餐许可范围的 ttl”，而是“根本没有设置用户在 HCL 中声明的 ttl，使用了 alidns API 默认的 600 秒作为 ttl 值”。

这个 bug 非常影响公司的自动化 IT 流程，希望能引起重视，尽快修复，谢谢。

**Comment by xiaozhu36** [comment]
@andyzhang495 收到，马上解决

**Comment by xiaozhu36** [comment]
HI @andyzhang495 Thanks for your feedback. PR #800 has fixed this issue.

**Comment by andyzhang495** [comment]
@xiaozhu36 wow, thanks for your fast response, really remarkable.

**Comment by xiaozhu36** [comment]
Fixed by 1.32.0
